submarine-server/server-submitter/submitter-yarn/src/main/java/org/apache/submarine/server/submitter/yarn/YarnUtils.java

```diff
@@ -33,7 +33,7 @@
 import java.util.List;
 import java.util.Map;
 
-import static org.apache.submarine.commons.runtime.conf.SubmarineConfiguration.SUBMARINE_RUNTIME_APP_TYPE;
+import static org.apache.submarine.commons.utils.SubmarineConfiguration.SUBMARINE_RUNTIME_APP_TYPE;
```

Comment:
maybe `SUBMARINE_RUNTIME_CLASS` is better?

Reply:
SUBMARINE_RUNTIME_APP_TYPE is another configuration. It defines the application type in the yarn. With it, Yarn UI show job type as submarine not tony.
This configuration is added in #24